docs/nlu/using-only-nlu.rst

```diff
@@ -0,0 +1,54 @@
+:desc: Find out how to use only Rasa NLU as a standalone NLU service for your chatbot or virtual assistant.
+
+.. _using-only-nlu:
+
+Using NLU Only
+==============
+
+
+If you want to use Rasa only as an NLU component, you can!
+
+Training NLU-only models
+------------------------
+
+To train an NLU model only, run:
+
+.. code-block:: bash
+
+   rasa train nlu
+
+This will look for NLU training data files in the ``data/`` directory
+and saved a trained model in the ``models/`` directory. 
+The name of the model will start with ``nlu-``.
+
+Testing your NLU model on the command line
+------------------------------------------
+
+To try out your NLU model on the command line, use the ``rasa shell`` command,
```

Comment:
To make it more explicit we should use `rasa shell nlu` here. If no NLU model is found this command fails, whereas `rasa shell` would still work if instead an Core model is found.